code/modules/antagonists/changeling/changeling.dm

```diff
@@ -96,6 +97,24 @@
 
 	/// A list of all memories we've stolen through absorbs.
 	var/list/stolen_memories = list()
+	
+	///	Keeps track of the currently selected profile.
+	var/datum/changeling_profile/current_profile
+	
+	/*	
+	 *	Static list of the quirks that you can gain/lose depending on who you shapeshift into.
+	 *	It'd be easiest to just get all of your target's quirks, but it wouldn't make sense for
+	 *	quirks like pacifist, brain tumor or paraplegic.
+	 */
+	var/list/mimicable_quirks_list = list(
```

Comment:
Ah, I wasn't sure what "Static" actually meant, I assumed it's a list that doesn't change. Updated the comment.

I did initially consider adding a var to the quirks themselves, but I wanted to avoid "sticking more changeling code everywhere" as you said it yourself. If you think it'd be a better solution, I can make the changes.

Reply:
Oh that's interesting, thank you for the explanation!

I kind of started off with the assumption that even if it was a var in the quirk datum, no one would bother making sure to change/set the var when adding quirks. (Hence someone would still have to maintain it themselves, and a list would be easier for that than looking through each quirk.)

I see what you mean though. I'll be gone for a bit but I'll change it to a var in the quirk datum when I get the time.

Reply:
Going to scrap the whole quirk thing for now because I'm not sure how to balance it.